Helpers/CommandConversion.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BrackeysBot.Helpers
+{
+    static class CommandConversion
+    {
+        const string SlashID = @"<87126>";
```

Comment:
No, not really. This string just can't conflict with anything that can exist in a message.

Reply:
Then why choose an arbitrary number instead of something like <slash>? Or the ASCII index of '/'? Just something that makes sense if you were to just see the converted message in plaintext,